pkg/controller/upload-controller.go

```diff
@@ -64,170 +69,176 @@ const (
 	uploadProxyCASecret     = "cdi-upload-proxy-ca-key"
 	uploadProxyServerSecret = "cdi-upload-proxy-server-key"
 	uploadProxyCAName       = "proxy.upload.cdi.kubevirt.io"
+
+	// UploadSucceededPVC provides a const to indicate an import to the PVC failed
+	UploadSucceededPVC = "UploadSucceeded"
 )
 
-// UploadController members
-type UploadController struct {
-	client                                    kubernetes.Interface
-	cdiClient                                 clientset.Interface
-	queue                                     workqueue.RateLimitingInterface
-	pvcInformer, podInformer, serviceInformer cache.SharedIndexInformer
-	pvcLister                                 corelisters.PersistentVolumeClaimLister
-	podLister                                 corelisters.PodLister
-	serviceLister                             corelisters.ServiceLister
-	pvcsSynced                                cache.InformerSynced
-	podsSynced                                cache.InformerSynced
-	servicesSynced                            cache.InformerSynced
-	uploadServiceImage                        string
-	pullPolicy                                string // Options: IfNotPresent, Always, or Never
-	verbose                                   string // verbose levels: 1, 2, ...
-	uploadProxyServiceName                    string
+// UploadReconciler members
+type UploadReconciler struct {
+	Client                 client.Client
+	CdiClient              cdiclientset.Interface
+	K8sClient              kubernetes.Interface
```

Comment:
Can't we get rid of `CdiClient` and `K8sClient`?

Reply:
confirming that you are getting rid of `CdiClient` and `K8sClient` in subsequent prs

Reply:
Yes, that is the plan.